### PR TITLE
#87 - Added S3Storage.value() method implementation

### PR DESCRIPTION
**Pull request by olegmoz**
Part of issue #87 
Added implementation for Storage.value() method in S3 storage.

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 PLease assign me

File: src/test/java/com/artipie/asto/S3StorageTest.java [hunk 1]
```diff
@@ -68,6 +71,17 @@ void shouldUploadObjectWhenSave(final AmazonS3 client) throws Exception {
...
 
+    @Test
+    void shouldGetObjectWhenLoad(final AmazonS3 client) {
+        final String bucket = UUID.randomUUID().toString();
+        client.createBucket(bucket);
+        final byte[] data = "data".getBytes();
+        final String key = "some/key";
+        client.putObject(bucket, key, new ByteArrayInputStream(data), new ObjectMetadata());
+        final byte[] value = new BlockingStorage(this.storage(bucket)).value(new Key.From(key));
+        MatcherAssert.assertThat(value, Matchers.equalTo(data));
```
Comment (paulodamaso) [inline src/test/java/com/artipie/asto/S3StorageTest.java]: @olegmoz Please add assertion failure message and do not use static matchers. You can also inline value variable into the assertion.

**Comment by 0crat** [comment]
This pull request [#97](https://github.com/artipie/asto/issues/97) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/5ed884fb-4887-4068-827a-315a92290aa8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by olegmoz** [comment]
@paulodamaso yes, of course implementation of other methods is coming. Made changes by your comment, please review

**Comment by paulodamaso** [comment]
@olegmoz Great, thanks!

**Comment by paulodamaso** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@olegmoz please fix merge conflicts

**Comment by olegmoz** [comment]
@g4s8 merged in master, please review

**Pull request merged**